### PR TITLE
Permit SMB as a protocol for Octopus MSI URL

### PR DESCRIPTION
**Pull request by hackjammer**
This is a requirement for an offline Octopus instance

### CODE REVIEW

**Comment by hackjammer** [comment]
@MJRichardson - thoughts on this? Can we merge it?

File: OctopusDSC/OctopusDSCHelpers.ps1 [hunk 1]
```diff
@@ -57,7 +67,11 @@ function Request-File {
             Write-Verbose "Downloading $url to $saveAs"
 
             try {
-                Invoke-WebClient -Url $url -OutFile $saveAs
+                switch ($downloadMethod)
```
Comment (matt-richardson) [inline OctopusDSC/OctopusDSCHelpers.ps1]: I dont think we need this switch block - `WebClient` can "download" from a network share.

Comment (matt-richardson) [reply OctopusDSC/OctopusDSCHelpers.ps1]: If this is removed, I think we can combine the two `switch` statements into one - we wont need the `$downloadMethod` variable.

**Comment by hackjammer** [comment]
You're right - didn't realise Invoke-WebClient actually supported UNC paths already - probably not worth merging this!

**Pull request closed without merging**